tdrs-backend/tdpservice/search_indexes/documents.py

```diff
@@ -0,0 +1,375 @@
+
+            'county_fips_code',
+            'stratum',
+            'zip_code',
+            'funding_stream',
+            'new_applicant',
+            'nbr_of_family_members',
+            'family_type',
+            'receives_sub_housing',
+            'receives_medical_assistance',
+            'receives_food_stamps',
+            'amt_food_stamp_assistance',
+            'receives_sub_cc',
+            'amt_sub_cc',
+            'child_support_amount',
+            'family_cash_recources',
+            'cash_amount',
+            'nbr_months',
+            'cc_amount',
+            'children_covered',
+            'cc_nbr_of_months',
+            'transp_amount',
+            'transp_nbr_months',
+            'transition_services_amount',
+            'transition_nbr_months',
+            'other_amount',
+            'other_nbr_of_months',
+            'sanc_reduction_amount',
+            'work_req_sanction',
+            'family_sanct_adult',
+            'sanct_teen_parent',
+            'non_cooperation_cse',
+            'failure_to_comply',
+            'other_sanction',
+            'recoupment_prior_ovrpmt',
+            'other_total_reductions',
+            'family_cap',
+            'reductions_on_receipts',
+            'other_non_sanction',
+            'waiver_evalu_control_grps',
+            'family_exempt_time_limits',
+            'family_new_child',
+            'blank',
+        ]
+
+
+@registry.register_document
+class T2DataSubmissionDocument(Document):
+    """Elastic search model mapping for a parsed T2 data file."""
+
+    class Index:
+        """ElasticSearch index generation settings."""
+
+        name = 't2_submissions'
+        settings = {
+            'number_of_shards': 1,
+            'number_of_replicas': 0,
+        }
+
+    class Django:
+        """Django model reference and field mapping."""
+
+        model = T2
+        fields = [
+            'record',
+            'rpt_month_year',
+            'case_number',
+            'disposition',
+            'fips_code',
+
+            'family_affiliation',
+            'noncustodial_parent',
+            'date_of_birth',
+            'ssn',
+            'race_hispanic',
+            'race_amer_indian',
+            'race_asian',
+            'race_black',
+            'race_hawaiian',
+            'race_white',
+            'gender',
+            'fed_oasdi_program',
+            'fed_disability_status',
+            'disabled_title_xivapdt',
+            'aid_aged_blind',
+            'receive_ssi',
+            'marital_status',
+            'relationship_hoh',
+            'parent_minor_child',
+            'needs_pregnant_woman',
+            'education_level',
+            'citizenship_status',
+            'cooperation_child_support',
+            'months_fed_time_limit',
+            'months_state_time_limit',
+            'current_month_state_exempt',
+            'employment_status',
+            'work_eligible_indicator',
```

Comment:
@jtimpe this data element does not exist for Tribal TANF T2 records, as shown [here](https://www.acf.hhs.gov/sites/default/files/documents/ofa/tribal_layout_for_section1.pdf). again, this is for awareness since i presume we're only focused on TANF. And there will be follow-ons for SSP-MOE and Tribal TANF? Note that this might affect files for pilot users in v3+, since some are tribal grantees.